app/styles/play/level.sass

```diff
@@ -259,6 +259,10 @@ $level-resize-transition-time: 0.5s
     right: 15px
     font-size: 30px
 
+  .ad-container
+    width: 100%
+    height: 90px
+    text-align: center
```

Comment:
This might make the interface really frustrating for players with short screens, since the aspect ratio has already kind of been engineered to just barely leave room for the bottom controls at common small laptop screen sizes. Like, instead of seeing this (1376 x 768):

<img width="1376" alt="screenshot 2016-03-15 17 07 57" src="https://cloud.githubusercontent.com/assets/99704/13798158/a84cbaa8-ead0-11e5-9288-0ac7b1820cba.png">

... they will see this:

<img width="1376" alt="screenshot 2016-03-15 17 08 09" src="https://cloud.githubusercontent.com/assets/99704/13798160/ae55a298-ead0-11e5-991c-1b43a6e537a0.png">

... and that's assuming they have minimal browser chrome and aren't being hit by the dock or some such bottom bar. This makes me wonder whether we shouldn't try something a little more fancy like only showing ads after level X or not showing them on levels when the screen is too small for it to be a good experience (or showing them on the bottom then). What do you think?